project-lifecycle.gv

```diff
@@ -0,0 +1,16 @@
+digraph lifecycle {
+  node [shape=box]
+  EndOfLife [label="End of Life"]
+  //rank=same; Proposal; Incubation; Graduated; Dormant; Deprecated; EndOfLife;
+  Proposal -> Incubation
+  Incubation [color=purple];
+  Incubation -> Graduated [color = purple];
+  Graduated [color=blue];
+  Graduated -> Dormant [color = blue];
+  Dormant [color=red];
+  Dormant -> Deprecated [color = red];
+  Deprecated -> EndOfLife;
```

Comment:
Thinking out loud here, can we say a project that is deprecated can be revived with a new proposal. I guess that is in line with what TSC thinks.